scripts/histmakers/mz_dilepton.py

```diff
@@ -188,6 +190,11 @@ def build_graph(df, dataset):
     axes = nominal_axes
     cols = nominal_cols
 
+    if args.addRunAxis and dataset.is_data:
+        run_edges = common.run_edges
+        axes = [*axes, hist.axis.Variable(run_edges+0.5, name = "run", underflow=False, overflow=False)]
```

Comment:
Is this something we might be asked also for the other analyses? If so, it might go in common.py, and could become a possibility everywhere (but can also be done later)